internal/services/apimanagement/api_management_logger_resource.go

```diff
@@ -70,10 +70,20 @@ func resourceApiManagementLogger() *pluginsdk.Resource {
 
 						"connection_string": {
 							Type:         pluginsdk.TypeString,
-							Required:     true,
+							Optional:     true,
 							Sensitive:    true,
 							ValidateFunc: validation.StringIsNotEmpty,
 						},
+						"endpoint_address": {
```

Comment:
Looking through the provider it seems the convention here is the call this 
```suggestion
						"endpoint_uri": {
```